DAWdleUnitTesting/UnitTesting.cpp

```diff
@@ -811,4 +811,64 @@ namespace MemoryTests {
 		char buffer2[10] = "hello";
 		EXPECT_EQ(memcmp(buffer1, buffer2, 3), 0);
 	}
-}
\ No newline at end of file
+}
+
+namespace ByteBuffer {
+
+	const StrA str = "dawdleDotcom"sa;
+
+	TEST(ByteBuff, Wrap) {
+		ByteBuf test;
+		U8 testValue = 'a';
+		U8* testdata = &testValue;
+		U32 dataSize = sizeof(testdata);
+		test.wrap(testdata, dataSize);
+		EXPECT_EQ(test.bytes, 'a');
+	}
+
+	TEST(ByteBuff, has_data_left) {
+		ByteBuf test;
+		U8 testValue = 'a';
+		U8* testdata = &testValue;
+		U32 dataSize = sizeof(testdata);
+		test.wrap(testdata, dataSize);
+		EXPECT_TRUE(test.has_data_left(dataSize));
```

Comment:
While good for a start, I would add a little more logic to help ensure has_data_left works correctly. For example, wrap a larger buffer, test that it has the full amount left, read n bytes, then test that it has bufferSize - n bytes left and that it doesn't have bufferSize - n + 1 bytes left.